stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/transactions.py

```diff
@@ -178,10 +256,13 @@ def bulk_sync(self, processed_items):
         ]
         helpers.bulk(self.client, actions)
 
-    def bulk_item_insert(self, items: Items, **kwargs) -> str:
+    @overrides
+    def bulk_item_insert(
+        self, items: Items, chunk_size: Optional[int] = None, **kwargs
+    ) -> str:
         """Bulk item insertion using es."""
         transactions_client = TransactionsClient()
-        transactions_client._create_item_index()
+        transactions_client.create_item_index()
```

Comment:
renamed this method to remove the leading `_` because this is a usage outside the class.